examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Suggestions:</p><ul><li>Remove "(which could be sparse or dense ...)" as I personally find that using parentheses distracting to read in most cases and here I don't think we gain much</li><li>Remove the <code>sample=0</code>  line and just fix the sample index the <code>plot(...)</code>  method</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done. The sparse or dense note is moved to below.</p>